src/js/components/user-management/__snapshots__/userform.test.js.snap

```diff
@@ -1,109 +1,3 @@
t-root MuiInput-underline MuiInputBase-formControl MuiInput-formControl\\" required=\\"\\" style=\\"width: 400px; max-width: 100%;\\"><input aria-invalid=\\"false\\" id=\\"email\\" name=\\"email\\" placeholder=\\"Email\\" required=\\"\\" type=\\"text\\" class=\\"MuiInputBase-input MuiInput-input\\" value=\\"\\"></div><p class=\\"MuiFormHelperText-root\\" id=\\"component-error-text\\"></p></div><div id=\\"password-holder\\" class=\\"edit-pass margin-top-small\\"><div class=\\"MuiFormControl-root required\\"><label class=\\"MuiFormLabel-root MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated\\" data-shrink=\\"false\\" for=\\"password\\">Password</label><div class=\\"MuiInputBase-root MuiInput-root MuiInput-underline MuiInputBase-formControl MuiInput-formControl MuiInputBase-adornedEnd\\" style=\\"width: 400px;\\" required=\\"\\"><input aria-invalid=\\"false\\" id=\\"password\\" name=\\"password\\" required=\\"\\" type=\\"password\\" class=\\"MuiInputBase-input MuiInput-input MuiInputBase-inputAdornedEnd\\" value=\\"\\"><div class=\\"MuiInputAdornment-root MuiInputAdornment-positionEnd\\"><button class=\\"MuiButtonBase-root MuiIconButton-root\\" tabindex=\\"0\\" type=\\"button\\"><span class=\\"MuiIconButton-label\\"><svg class=\\"MuiSvgIcon-root\\" focusable=\\"false\\" viewBox=\\"0 0 24 24\\" aria-hidden=\\"true\\" role=\\"presentation\\"><path d=\\"M12 7c2.76 0 5 2.24 5 5 0 .65-.13 1.26-.36 1.83l2.92 2.92c1.51-1.26 2.7-2.89 3.43-4.75-1.73-4.39-6-7.5-11-7.5-1.4 0-2.74.25-3.98.7l2.16 2.16C10.74 7.13 11.35 7 12 7zM2 4.27l2.28 2.28.46.46C3.08 8.3 1.78 10.02 1 12c1.73 4.39 6 7.5 11 7.5 1.55 0 3.03-.3 4.38-.84l.42.42L19.73 22 21 20.73 3.27 3 2 4.27zM7.53 9.8l1.55 1.55c-.05.21-.08.43-.08.65 0 1.66 1.34 3 3 3 .22 0 .44-.03.65-.08l1.55 1.55c-.67.33-1.41.53-2.2.53-2.76 0-5-2.24-5-5 0-.79.2-1.53.53-2.2zm4.31-.78l3.15 3.15.02-.16c0-1.66-1.34-3-3-3l-.17.01z\\"></path></svg></span><span class=\\"MuiTouchRipple-root\\"></span></button></div></div><p class=\\"MuiFormHelperText-root\\" id=\\"component-error-text\\"></p></div><div><div class=\\"help-text\\" id=\\"pass-strength\\">Strength: <meter max=\\"4\\" min=\\"0\\" value=\\"\\" high=\\"3.9\\" optimum=\\"4\\" low=\\"2.5\\"></meter></div><div class=\\"pass-buttons\\"><button class=\\"MuiButtonBase-root MuiButton-root MuiButton-text MuiButton-textPrimary\\" tabindex=\\"0\\" type=\\"button\\"><span class=\\"MuiButton-label\\">Generate</span><span class=\\"MuiTouchRipple-root\\"></span></button></div></div></div><div class=\\"flexbox\\" style=\\"justify-content: flex-end; margin: 24px 0px -16px 0px;\\"><button class=\\"MuiButtonBase-root MuiButton-root MuiButton-contained Mui-disabled Mui-disabled\\" tabindex=\\"-1\\" type=\\"button\\" disabled=\\"\\" id=\\"submit_button\\"><span class=\\"MuiButton-label\\">Create user</span></button></div></form></div><div class=\\"MuiDialogActions-root MuiDialogActions-spacing\\"></div></div></div><div tabindex=\\"0\\" data-test=\\"sentinelEnd\\"></div></div>"`;
```

Comment:
maybe this line can be longer?
ok, can we maybe make it a bit more editor-friendly?

Reply:
that is an autogenerated snapshot - if that would be modified the corresponding test fails... the format depends on the renderer that is used to generate the snapshot, which in this case has to be a different one than what it was previously, as I encapsulated the userform in its dialog in order to decouple this from the overall usermanagement component...